README.md

```diff
@@ -6,3 +6,5 @@
 ### Authors
```

Comment:
- [ ] The README file is the first file a visitor sees when viewing your project, let's add the missing parts to make it more professional, here's a [template](https://github.com/microverseinc/readme-template/blob/master/README.md) you can use.